<!DOCTYPE html>

<html>

<body> 
  <h1>
    Jarrett D. Phillips (He/Him/His)
  </h1>
  <h3>
    <strong> BSc. (Hons.), MBinf., PhD. </strong>
    <br> Adjunct Professor
    <br> School of Computer Science
    <br> Department of Integrative Biology
    <br> University of Guelph
    </h3>
  <img src = "img.jpg" width = 300 height = 400>
  <h2>
    Professional Summary of Research Interests and Expertise
    </h2>
  <p>
    I am a highly motivated and passionate bioinformatician, data scientist and statistician naturally driven by curiosity 
    to use mathematical, statistical and computational methods to answer fundamental and applied research questions 
    in biodiversity science, evolutionary biology, ecology, genomics and bioinformatics, particularly related to 
    molecular species identification and discovery through DNA barcoding, environmental DNA (eDNA) and other DNA-based approaches. 

    My academic work and research interests can best be described as computational molecular biodiversity science. Biodiversity 
    is under threat in a rapidly changing world, where mitigation requires innovative and collaborative solutions from multiple 
    disciplines. DNA-based specimen identification and species discovery through techniques like DNA barcoding and environmental 
    DNA (eDNA) offer promising ways forward, yet produce overwhelming amounts of data. I leverage AI/ML/Data Science/Big Data 
    methods to help researchers find meaningful signal in a vast sea of noise.
</p>
    <h2>
    Recent News
    </h2>
  <ul>
    <li>I received a $40000 CAD Food from Thought Advancing Research Impact Fund (ARIF) Livestock Innovation grant to develop the Dynamic Population Model (DPM) as part of my work with GBADs (13/06/2024) </li>
    <li>Two book chapters on DNA barcoding for specimen identification and species delimitation have now been published online by Springer Nature (30/04/2024) </li>
    <li>A preprint on GBADs informatics strategy, data quality, and model interoperability is now available </li>
    <li>My preprint on statistical modelling of seafood mislabelling in Canada is now on bioRxiv (08/02/2024)</li>
    <li>I attended the GBADs Technical Workshop in Liverpool, England from 11/12/2023 to 15/12/2023</li>
    <li>I was appointed Adjunct Professor in the School of Computer Science at the University of Guelph on 23/11/2023</li>
    <li>I joined the Global Burden of Animal Diseases (GBADs) Informatics team at the University of Guelph on 22/06/2023</li>
    <li>The VLF paper was published on 26/01/23</li>
    <li>My paper introducing and outlining the VLF R package has been accepted for publication in the <i>Biodiversity Data Journal</i> on 29/11/2022</li>
    <li>I received $30000.00 CAD in funding from the Food from Thought Advancing Research Impact Fund (ARIF) to develop a Bayesian hierarchical binary logistic time-series regression model of seafood fraud in the Canadian supply chain on 20/10/2022</li>
    <li>My recent paper arguing a lack of statistical rigor in DNA barcoding was featured as a University of Guelph College of Physical and Engineering Sciences (CEPS) Research Highlights article on 15/08/2022</li>
    <li> My opinion paper on statistical aspects of DNA barcoding and the DNA barcode gap was published in <i>Frontiers in Ecology and Evolution</i> on 14/04/2022</li>
  </ul>
  <h2>
    Research Interests
    </h2>
  <ul>
    <li> Agent-based and individual-based modelling </li>
    <li> Bayesian computing and statistics in R and Stan </li>
    <li> Biodiversity informatics (<it>e.g.</it>, DNA barcoding, environmental DNA (eDNA), DNA sequences analysis) </li>
    <li> Data science </li>
    <li> Machine learning (<it>e.g.</it>, association rule generation, clustering, classification, and dimensionality reduction of (e)DNA sequence data and related metadata) </li>
    <li> Nonparametric statistics (<it>e.g.</it>, bootstrap resampling, local regression, kernel methods) </li>
    <li> Spatiotemporal modelling (<it>e.g.</it>, conditionally autoregressive models (CARs), Gaussian processes (GPs)/Kriging, generalized additive models (GAMs), time series) </li>
    <li> Stochastic optimization (<it>e.g.</it>, random search algorithms, genetic algorithms (GAs), simulated annealing) </li>
   </ul>  
  <h2>
    Current Projects
    </h2>
<h3>
    Agent-based, equation-based, and individual-based models for animal disease burden
</h3>
    <ul>
      <li> We are currently developing an age- and sex-structured compartmentalized equation-based model in R to assess disease burden in livestock species such as cattle, small ruminants (<it>e.g.</it>, sheep and goats), and poultry in developing countries like Ethiopia </li>
    </ul>
  <h3>
    Theoretical Aspects of DNA-based Taxon Identification
    </h3>
  <ul>
    <li> Developing methods to estimate likely required specimen sample sizes for genetic diversity assessment within species using DNA barcoding </li>
    <ul>
      <li> Practical sample sizes for DNA barcoding typically range between 5-10 indviduals per species, but required levels of sampling depth are highly 
      dependent on the evolutionary history and ecology of the species under study, as well as species rarity and overall project costs. </li>
      <li> I created <a href = "https://cran.r-project.org/web/packages/HACSim/index.html">HACSim</a> (Haplotype Accumulation Curve Simulator), a novel nonparametric stochastic (Monte Carlo) local search optimization algorithm, which has been 
      developed to better estimate likely required specimen sample sizes based on asymptotic behaviour seen in species' haplotype accumulation curves. </li>
      <li> HACSim has been shown to work well for a variety of species of socioeconomic relevance such as fishes, insects and arachnids based on extensive 
      simulation studies. </li>
      <li> The publicatiion of HACSim in <it></it>PeerJ Computer Science</it> was one of the Top-5 most viewed articles in the category "Optimization Theory and Compution"</li>
         </ul>
    <li> Developing methods for better visualization and inference of the DNA barcode gap
      <ul>
        <li> The DNA barcode gap &#8212; the difference between genetic variation observed with and among species, is most often visualized using histograms; however, 
        these plots can be misleading due to dependence on user-defined parameters, thus greatly affecting the overall shape of probability distributions. </li> 
        <li> I propose Kernel density estimation provides a better path forward when it comes to establishing the efficacy of DNA barcoding as a molecular identification
        tool. </li>
        <li> I also propose the use of nonparametric bootstrapping, specifically the <i>m</i>-out-of-<i>n</i> boostrap, which can be used to estimate the sampling distributions of quantities 
        of interest in DNA barcoding, particularly extreme order statistics like the minimum interspecific and maximum intraspecific genetic distances, in addition 
        to the DNA barcode gap. </li>
        <li>We are currently developing a population genetic model of the DNA barcode gap based on the Multispecies Coaleacent (MSC) </li>
      </ul>
    </ul>
  <h3>
    Building the Reference Library of Life
    </h3>
  <ul>
    <li> Constructing the largest DNA barcode reference sequence library for North American butterflies </li>
    <ul>
      <li> Butterflies are some on the most genetically and morpohologically diverse insects on the planet. One aspect of my work involves generating a DNA barcode 
        library for over 97% (>800 species) of known butterfly species found in North America </li> 
      <li> Current work using this dataset involves correlating species' genetic diversity with geographical covariates (such as latitude and longitude) 
      using semiparametric and nonparametric regression approaches including Generalized Additive Models (GAMs) and Local Regression (LOESS) </li>
     </ul>
     </ul>
  <h3> 
     Development of R Software Packages and R Shiny Web Apps for Molecular Biodiversity Assessment
    </h3>
  <ul>
    <li> HACSim is available for download as an R package through the Comprehensive R Archive Network (CRAN) and as an R Shiny web application at <a href = "https://jphill01.shinyapps.io/HACSim/">shinyapps.io</a>
       </li>
  <li> <a href = "https://cran.r-project.org/web/packages/VLF/index.html">VLF</a> is an R package to detect very low frequency variants (VLFs), such as sequencing and PCR errors, and compute error rates at second codon positions in DNA sequences. The package is on CRAN.
  </li>
    </ul>
  <h3>
    Assessing R Metadata Reporting and Standards in the Ecological Literature
  </h3>
    <ul>
      <li>
        Within many scientific publications that employ R, lack of proper citation of R versions and packages employed within analyses is rampant
      </li>
      <li>
        We mine article metadata spanning five different ecological journals for publications appearing in 2019 and find significant variation in how R is currently being reported
      </li>
      <li>
        We propose a simple way of standardizing reporting that will help mitigate biases in future studies
      </li>
    </ul>
  </h3>
  <h2>
    Student Supervision and Mentorship
  </h2>
  <ul>
    <li> Richard Cui (with Dan Gillis) -- University of Guelph -- Summer Undergraduate Research Assistant (URA) -- Dynamical modelling of seafood fraud in the supply chain </li>
    <li>Nikolett Toth (2025) -- University of Guelph --  CIS*4910 -- Mining association rules for eDNA spatiotemporal sampling </li>
    <li>Nikolett Toth (with Dan Gillis, 2024) -- University of Guelph --  CIS*4900 -- Mining association rules for eDNA spatiotemporal sampling </li>
    <li>Nikolett Toth (with Dan Gillis, 2024) -- University of Guelph -- Summer Undergraduate Research Assistant (URA) -- Mining association rules for eDNA spatiotemporal sampling </li>
    <li>Nathan Zeinstra (with Dirk Steinke, 2024) -- University of Guelph -- IBIO*6070 -- Habitat occupancy modelling of sea lamprey environmental DNA </li>
    <li> Fynn De Vuono-Fraser (with Dan Gillis, 2023) -- University of Guelph -- CIS*4900 -- Bayesian modelling of seafood fraud in the Canadian supply chain</li>
    <li> Zaid Al-Gayyali (with Dan Gillis, 2023)	-- University of Guelph -- Summer Undergraduate Research Assistant (URA) -- Seafood Fraud Visualization Tool R Shiny web app </li>
    <li> Fynn De Vuono-Fraser (with Dan Gillis, 2023) -- University of Guelph -- STAT*4600 -- Bayesian modelling of seafood fraud in the Canadian supply chain </li>
    <li> Amina Asif (with Bob Hanner, 2022) -- University of Guelph -- BINF*6999 -- DNA barcode gap analysis of Canadian agricultural pests and disease vectors </li>
    <li> Navdeep Singh (with Dan Gillis, 2021) -- University of Guelph CIS*4900  -- HACSim RShiny web application </li>
    <li> Maya Persram (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Ashley Chen (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Olivia Friesen Kroeker (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Scarlett Bootsma (with Dan Gillis, 2020-2021) -- University of Guelph -- CIS*4900/4910 -- HACSim simulation study </li>
    <li> Danielle St. Jean (with Dan Gillis, 2018-2019) -- University of Guelph -- MSc. thesis (Math) -- DNA barcode sequence classification with machine learning </li>
    <li> Christina Fragel (with Bob Hanner, 2018-2019) -- University of Guelph -- BINF*6999 -- DNA barcode sequence classification with machine learning </li>
    <li> Jiaojia (Paula) Yu (with Bob Hanner, 2018-2019) -- University of Guelph -- BINF*6999  MDMAPR qPCR R Shiny app	</li>
    <li> Steven French (with Dan Gillis, 2018) -- University of Guelph -- CIS*4900/4910 -- HACSim R package </li>
    <li> Julia Harvie (with Bob Hanner, 2018-2019) -- University of Guelph --  MCB*4500/4510 --  Data mining GenBank and BOLD </li>
    <li> Ankita Bhanderi (with Bob Hanner, 2018) -- University of Guelph -- BINF*6999 -- Data mining GenBank and BOLD </li>
  </ul>
  <h2>
    Research Collaborators
  </h2>
  <ul>
    <li> Dr. Daniel Gillis (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Robert Hanner (Biodiversity Institute of Ontario, Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Robert Young (Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Jacopo D'Ercole (Centre for Biodiversity Genomics, University of Guelph, Canada) </li>
    <li> Dr. Cortland Griswold (Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Nicolas Hubert (Institut de Recherche pour le Dévelopement, Université de Montpellier France) </li>
    <li> Dr. Dirk Steinke (Centre for Biodiversity Genomics and Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Luiza Antonie (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Deborah Stacey (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Theresa Bernardo (Department of Population Medicine, University of Guelph, Canada) </li>
    <li> Dr. Kurtis Sobkowich (Department of Population Medicine, University of Guelph, Canada) </li>
    <li> Dr. Le Nguyen (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Mike Yodzis (Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Ed Sykes (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Elif Acar (Department of Mathematics and Statistics, University of Guelph, Canada) </li>
    <li> Dr. Stefan Kremer (School of Computer Science, University of Guelph, Canada) </li>
    <li> Dr. Graham Taylor (School of Engineering, University of Guelph, Canada) </li>
    <li> Dr. Prasad Daggupati (School of Engineering, University of Guelph, Canada) </li>
    <li> Dr. Zeny Feng (Department of Mathematics and Statistics, University of Guelph, Canada) </li>
    <li> Dr. Allan Willms (Department of Mathematics and Statistics, University of Guelph, Canada) </li>
    <li> Dr. Richard Heck (School of Environmental Science, University of Guelph, Canada) </li>
    <li> Dr. Shoshanah Jacobs (Biodiversity Institute of Ontario, Department of Integrative Biology, University of Guelph, Canada) </li>
    <li> Dr. Aicheng Chen (Department of Chemistry, University of Guelph, Canada) </li>
  </ul>
  <h2>
    Funding
  </h2>
  <!-- <img src="UofG_Cornerstone_rgb.png", width="100" height="90"> -->
  <!-- <img src="CFREF_Logos-2.png" width="200" height="100"> -->
  <img src="comb logos.png" width="500" height="100">
  <h2>
    Curriculum Vitae 
    </h2>
  <a href = "CV June 2025 - Phillips, Jarrett.pdf">CV</a> &#8212 last updated June 2025
  <h2>
    Contact
  </h2>
      <a href = "mailto: jphill01@uoguelph.ca">GryphMail</a>
      <br> 
      <a href = "mailto: phillipsjarrett1@gmail.com">GMail</a>
  <h2>
    Relevant Links
    </h2>
    <a href = "https://www.uoguelph.ca/computing/people/jarrett-phillips">School of Computer Science (SoCS)</a> 
  <br>
   <br>
  <a href = "https://github.com/jphill01">GitHub</a> 
  <br>
  <a href = "https://scholar.google.ca/citations?user=2YE-Y4cAAAAJ&hl=en">Google Scholar</a>
  <br>
  <a href = "https://www.linkedin.com/in/jarrettdphillips/">LinkedIn</a>
  <br>
  <a href = "https://orcid.org/0000-0001-8390-386X">ORCiD</a>
  <br>
  <a href = "https://www.researchgate.net/profile/Jarrett-Phillips">ResearchGate</a>
  <br>
  <a href = "https://twitter.com/JarrettDPhilli1">Twitter/X</a>
  
</body>
  
</html>
