<!DOCTYPE html>

<html>

<body> 
  <h1>
    Jarrett D. Phillips (He/Him/His)
  </h1>
  <h3>
    <strong> BSc. (Hons.), MBinf., PhD. </strong>
    <br> School of Computer Science
    <br> Department of Integrative Biology
    <br> Centre for Biodiversity Genomics
    <br> Biodiversity Institute of Ontario
    <br> University of Guelph
    </h3>
  <img src = "img.jpg" width = 300 height = 400>
  <h2>
    Professional Summary of Research Interests and Expertise
    </h2>
  <p>
    I'm a highly motivated and passionate bioinformatician, data scientist and researcher naturally driven by curiosity 
    to use mathematical, statistical and computational methods to answer fundamental and applied research questions 
    in biodiversity science, evolutionary biology, ecology, genomics and bioinformatics, particularly related to 
    molecular species identification and discovery through DNA barcoding and other DNA-based approaches. 
    </p>
  <h2>
    Current Projects
    </h2>
  <h3>
    Theoretical Aspects of DNA-based Taxon Identification
    </h3>
  <ul>
    <li> Developing methods to estimate likely required specimen sample sizes for genetic diversity assessment within species using DNA barcoding </li>
    <ul>
      <li> Practical sample sizes for DNA barcoding typically range between 5-10 indviduals per species, but required levels of sampling depth are highly 
      dependent on the evolutionary history and ecology of the species under study, as well as species rarity and overall project costs. </li>
      <li> I created <a href = "https://cran.r-project.org/web/packages/HACSim/index.html">HACSim</a> (Haplotype Accumulation Curve Simulator), a novel nonparametric stochastic (Monte Carlo) local search optimization algorithm, which has been 
      developed to better estimate likely required specimen sample sizes based on asymptotic behaviour seen in species' haplotype accumulation curves. </li>
      <li> HACSim has been shown to work well for a variety of species of socioeconomic relevance such as fishes, insects and arachnids based on extensive 
      simulation studies. </li>
         </ul>
    <li> Developing methods for better visualization and inference of the DNA barcode gap
      <ul>
        <li> The DNA barcode gap &#8212;, the difference between genetic variation observed with and among species, is most often visualized using histograms; however, 
        these plots can be misleading due to dependence on user-defined parameters, thus greatly affecting the overall shape of probability distributions. </li> 
        <li> I propose Kernel density estimation provides a better path forward when it comes to establishing the efficacy of DNA barcoding as a molecular identification
        tool. </li>
        <li> I also propose the use of nonparametric bootstrapping, specifically the <i>m</i>-out-of-<i>n</i> boostrap, which can be used to estimate the sampling distributions of quantities 
        of interest in DNA barcoding, particularly extreme order statistics like the minimum interspecific and maximum intraspecific genetic distances, in addition 
        to the DNA barcode gap. </li>
      </ul>
    </ul>
  <h3>
    Building the Reference Library of Life
    </h3>
  <ul>
    <li> Constructing the largest DNA barcode reference sequence library for North American butterflies </li>
    <ul>
      <li> Butterflies are some on the most genetically and morpohologically diverse insects on the planet. One aspect of my work involves generating a DNA barcode 
        library for over 97% (>800 species) of known butterfly species found in North America </li> 
      <li> Current work using this dataset involves correlating species' genetic diversity with geographical covariates (such as latitude and longitude) 
      using semiparametric and nonparametric regression approaches including Generalized Additive Models (GAMs) and Local Regression (LOESS) </li>
     </ul>
     </ul>
  <h3> 
     Development of R Software Packages and R Shiny Web Apps for Molecular Biodiversity Assessment
    </h3>
  <ul>
    <li> HACSim is available for download as an R package through the Comprehensive R Archive Network (CRAN) and as an R Shiny web application at <a href = "https://jphill01.shinyapps.io/HACSim/">shinyapps.io</a>
       </li>
  <li> <a href = "http://cran.nexr.com/web/packages/VLF/index.html">VLF</a> is an R package to detect very low frequency variants (VLFs), such as sequencing and PCR errors, and compute error rates at second codon positions in DNA sequences. The package is on CRAN.
  </li>
    </ul>
  <h3>
    Assessing R Metadata Reporting and Standards in the Ecological Literature
  </h3>
    <ul>
      <li>
        Within many scientific publications that employ R, lack of proper citation of R versions and packages employed within analyses is rampant
      </li>
      <li>
        We mine article metadata spanning five different ecological journals for publications appearing in 2019 and find significant variation in how R is currently being reported
      </li>
      <li>
        We propose a simple way of standardizing reporting that will help mitigate biases in future studies
      </li>
    </ul>
  </h3>
  <h2>
    Student Supervision and Mentorship
  </h2>
  <ul>
    <li> Amina Asif (with Bob Hanner, 2022) -- University of Guelph -- BINF*6999 -- DNA barcode gap analysis </li>
    <li> Navdeep Singh (with Dan Gillis, 2021) -- University of Guelph CIS*4900  -- HACSim RShiny web application </li>
    <li> Maya Persram (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Ashley Chen (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Olivia Friesen Kroeker (with Bob Hanner, 2020-present) -- University of Guelph Hanner Lab volunteer </li>
    <li> Scarlett Bootsma (with Dan Gillis, 2020-2021) -- University of Guelph -- CIS*4900/4910 -- HACSim simulation study </li>
    <li> Christina Fragel (with Bob Hanner, 2018-2019) -- University of Guelph -- BINF*6999 -- DNA barcode sequence classification with machine learning </li>
    <li> Jiaojia (Paula) Yu (with Bob Hanner, 2018-2019) -- University of Guelph -- BINF*6999  MDMAPR qPCR R Shiny app	</li>
    <li> Steven French (with Dan Gillis, 2018) -- University of Guelph -- CIS*4900/4910 -- HACSim R package </li>
    <li> Julia Harvie (with Bob Hanner, 2018-2019) -- University of Guelph--  MCB*4500/4510 --  Data mining GenBank and BOLD </li>
    <li> Ankita Bhanderi (with Bob Hanner, 2018) -- University of Guelph -- BINF*6999 -- Data mining GenBank and BOLD </li>
  </ul>
  <h2>
    Curriculum Vitae 
    </h2>
  <a href = "CV June 2022 - Phillips, Jarrett.pdf">CV</a> &#8212 last updated June 2022
  <h2>
    Contact
  </h2>
      <a href = "mailto: jphill01@uoguelph.ca">GryphMail</a>
      <br> 
      <a href = "mailto: phillipsjarrett1@gmail.com">GMail</a>
  <h2>
    Relevant Links
    </h2>
  <a href = "https://github.com/jphill01">GitHub</a> 
  <br>
  <a href = "https://scholar.google.ca/citations?user=2YE-Y4cAAAAJ&hl=en">Google Scholar</a>
  <br>
  <a href = "https://www.linkedin.com/in/jarrettdphillips/">LinkedIn</a>
  <br>
  <a href = "https://orcid.org/0000-0001-8390-386X">ORCiD</a>
  <br>
  <a href = "https://www.researchgate.net/profile/Jarrett-Phillips">ResearchGate</a>
  <br>
  <a href = "https://twitter.com/JarrettDPhilli1">Twitter</a>
  
</body>
  
</html>
